publication/src/test/scala/hmda/publication/reports/aggregate/A11XSpec.scala

```diff
@@ -0,0 +1,124 @@
+  val descriptionN116 = "Pricing information for VA refinancing loans, first lien, 1- to 4-family owner-occupied dwelling (excludes manufactured homes), by borrower or census tract characteristics"
+
+  "Generate an Aggregate 11-2 report" in {
+    A11_2.generate(aggregateSource, fips).map {
+      case AggregateReportPayload(reportId, fipsCode, report) =>
+        reportId mustBe "11-2"
+        fipsCode mustBe fips.toString
+        report.parseJson.asJsObject.getFields("table", "type", "description", "year", "msa") match {
+          case Seq(JsString(table), JsString(reportType), JsString(desc), JsString(reportYear), msa) =>
+            table mustBe "11-2"
+            reportType mustBe "Aggregate"
+            desc mustBe descriptionA112
+            msa.asJsObject.getFields("name") match {
+              case Seq(JsString(msaName)) => msaName mustBe "Corvallis, OR"
+            }
+        }
+    }
+  }
+
+  "Include correct borrower Characteristics" in {
+    A11_5.generate(aggregateSource, fips).map {
+      case AggregateReportPayload(reportId, fipsCode, report) =>
+        reportId mustBe "11-5"
+        fipsCode mustBe fips.toString
+        report.parseJson.asJsObject.getFields("borrowerCharacteristics") match {
+          case Seq(JsArray(characteristics)) =>
+            characteristics must have size 5
+            characteristics.head.asJsObject.getFields("characteristic", "races") match {
+
+              case Seq(JsString(char), JsArray(races)) =>
+                char mustBe "Race"
+                races must have size 8
+                races.head.asJsObject.getFields("race", "pricingInformation") match {
+
+                  case Seq(JsString(race), JsArray(pricing)) =>
+                    race mustBe "American Indian/Alaska Native"
+                    pricing must have size 10
+                }
+            }
+        }
+    }
+  }
+
+  "Include correct Census Tract Characteristics" in {
+    A11_5.generate(aggregateSource, fips).map { result =>
+      result.report.parseJson.asJsObject.getFields("censusTractCharacteristics") match {
+
+        case Seq(JsArray(characteristics)) =>
+          characteristics must have size 2
+          characteristics.head.asJsObject.getFields("characteristic", "compositions") match {
+
+            case Seq(JsString(char), JsArray(races)) =>
+              char mustBe "Racial/Ethnic Composition"
+              races must have size 5
+              races.head.asJsObject.getFields("composition", "pricingInformation") match {
+
+                case Seq(JsString(race), JsArray(pricing)) =>
+                  race mustBe "Less than 10% minority"
+                  pricing must have size 10
+              }
+          }
+      }
+    }
+  }
+
+  val lars2 = lar100ListGen.sample.get.map { lar: LoanApplicationRegister =>
+    val loan = lar.loan.copy(loanType = oneTo3, propertyType = 1, purpose = oneTo3, occupancy = 1)
```

Comment:
Best to update these values to exactly fit the filters for `N11_6`. We want the spec to run with as many relevant values as possible.

Reply:
I chose these values on purpose.  I don't want every single lar to be included in the report, I'd like the filters to be able to select a section of the dataset.